components/docs-chef-io/content/automate/ha_inplace_migration.md

```diff
@@ -126,6 +133,25 @@ AND
     path = "/mnt/automate_backups/elasticsearch"
 ```
 
+{{< note >}}
+
+In case `backup_config = "file_system"` had been provided in config.toml of Automate HA deployment, then please patch the below OpenSearch config from bastion before starting the restore.
```

Comment:
If `backup_config = "file_system"` had been provided in config.toml of Automate HA deployment, patch the below OpenSearch config from bastion before starting the restore.